src/config.coffee

```diff
@@ -1190,6 +1191,13 @@ Config.addSchemaEnforcers
 isPlainObject = (value) ->
   _.isObject(value) and not _.isArray(value) and not _.isFunction(value) and not _.isString(value) and not (value instanceof Color)
 
+sortObject = (value) ->
+  return value unless _.isObject(value) and not _.isArray(value)
```

Comment:
I think it'd be good to use the `isPlainObject` function here instead of `_.isObject and _.isArray`, because that function contains this same loigic, but it looks like it handles a few additional edge cases (e.g. `Color` objects).